### PR TITLE
10.6.x

### PR DESCRIPTION
**Pull request by CHBMB**
This is the v10.6.0 build of airsonic-advanced.  Builds locally on all archs.  x64 confirmed as working.  I *think* I've got the jenkins-vars.yml and readme-vars.yml correct.

### CODE REVIEW

**Comment by CHBMB** [comment]
Just a thought, looking over the airsonic-advanced repo. They have a dedicated `stable/v10.6.0` branch and `master` is actually their v11 development branch.  Might be worth rejigging things with branch names here to make it a bit more aligned with upstream?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ rm -rf \
+	/tmp/* \
+	/var/lib/apt/lists/* \
+	/var/tmp/*
+
+# add local files
+COPY root/ /
+
+# ports and volumes
+EXPOSE 4040
```
Comment (TheSpad) [inline Dockerfile:50]: Your indentation is a random mix of tabs and spaces, should be to the spec.

File: root/etc/services.d/airsonic/run [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	-Dairsonic.defaultMusicFolder=/music \
+	-Dairsonic.defaultPlaylistFolder=/playlists \
+	-Dairsonic.defaultPodcastFolder=/podcasts \
+	-Dairsonic.home="${AIRSONIC_ADVANCED_SETTINGS}" \
+	-Djava.awt.headless=true \
+	-Dserver.contextPath="${URL_BASE}" \
+	-Dserver.host=0.0.0.0 \
+	-Dserver.port=4040 \
+	"${RUN_ARRAY[@]}" \
+	-jar airsonic.war
```
Comment (TheSpad) [inline root/etc/services.d/airsonic/run:26]: Same with the indentation here

**Comment by CHBMB** [comment]
@TheSpad Good pickup.  I would love to blame the original airsonic repo, but there's every possibility it's me.  

I've actually been thinking about this overnight.  For continuity I think the way forward is:

v11 = master
v10.6.0 = stable/v10.6.0

That way it's congruent with upstream, however I think `latest` should point to the `stable/v10.6.0` branch to try and mitigate the inescapable fact that some people will just try and drop this in as a replacement to airsonic.  There's a chance that might work if it's the v10.6.0 branch and almost certain breakage going to v11.

**Pull request closed without merging**